autogpts/forge/forge/config/ai_directives.py

```diff
@@ -22,26 +20,7 @@ class AIDirectives(BaseModel):
     constraints: list[str] = Field(default_factory=list)
     best_practices: list[str] = Field(default_factory=list)
 
-    @staticmethod
-    def from_file(prompt_settings_file: Path) -> "AIDirectives":
-        from forge.logging.helpers import request_user_double_check
-
-        (validated, message) = validate_yaml_file(prompt_settings_file)
-        if not validated:
-            logger.error(message, extra={"title": "FAILED FILE VALIDATION"})
-            request_user_double_check()
```

Comment:
With this usage gone, iirc we can remove `request_user_double_check` from `forge`. Maybe it's still used in `autogpt` but then it should be defined there.

Reply:
When I remove `request_user_double_check`, then `user_friendly_output` is also not used, so removing 🧹
Also moved `print_attribute` and `speak` to `logging/utils.py`, so I could remove `logging/helpers.py`.

Reply:
if `user_friendly_output` is not used, then the `ConsoleFormatOutputLogger` or whatever it's called is also not used :)